docs/overview/index.md

```diff
@@ -0,0 +1,24 @@
+# Product Overview
+
+HCL DX is the trusted solution for an enterprise's digital experience at the intersection of processes, content and applications, built Cloud Native serving your customers, employees and partners. HCL DX is architected to build more than just websites. Deliver the functionality and stability your users need and expect.
+
+You can customize HCL DX to meet the needs of your organization, users, and user groups. You can adapt the look and feel of the site to fit the standards of your organization and to customize page content for users and groups in accordance with business rules and user profiles. Users, such as business partners, customers, or employees, can further customize their own views of the sites you build, if you let them. Users can add applications to pages and arrange them as they want. By aggregating portlets in one place and giving users the power to customize their own desktops, the HCL DX gives users a means for doing business efficiently and with high satisfaction.
+
+## For Practitioners
+
+HCL DX includes a range of tools for managing content and maintaining web and mobile digital experiences. Those sites can blend applications and content together, in a targeted way according to the end user.
+
+Tools for practitioners include content management, asset management, site building via Design Studio and Site Manager, personalization management and more.
+
+## For Developers
+
+HCL DX provides a wide range of APIs and tools to enable developers to customize the digital experience. Developers can use Script Applications to produce javascript based applications. Java Portlets written to the JSR168 or JSR286 specification can be deployed too.
+
+Developers can use REST and Java APIs to access all of HCL DX capability. The DXClient tool can be used to deploy and manage applications, and to manage servers.
+
+## For System Administrators
+
+HCL DX can be deployed in a traditional WebSphere cluster, or it can be deployed to a Kubernetes environment. HCL DX provides a range of scriptable interfaces so that you can automate management of your infrastructure.
```

Comment:
Can we use "WebSphere Application Server cluster"?

Reply:
updated to "IBM WebSphere Application Server Cluster". 
Added (r) symbol added to IBM WebSphere Application Server too